python/emitter.js

```diff
@@ -104,7 +104,15 @@
     }
     if (test.config.browsersInfo) {
         addHook('deps', 'from applitools.common.ultrafastgrid import DesktopBrowserInfo, IosDeviceInfo, ChromeEmulationInfo, ScreenOrientation')
-        addHook('beforeEach', python`    conf.add_browser(${{value: test.config.browsersInfo, type: 'BrowsersInfo'}})`)
+		addHook('beforeEach', python`    ${{value: test.config.browsersInfo, type: 'BrowsersInfo'}}`)
+    }
+	if (test.config.layoutBreakpoints) {
+        if (Array.isArray(test.config.layoutBreakpoints)) {
+            addHook('beforeEach', python`    conf.set_layout_breakpoints(${test.config.layoutBreakpoints[0]}, ${test.config.layoutBreakpoints[1]})`)
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8717150)
```suggestion
		addHook("beforeEach", python`    conf.set_layout_breakpoints(${test.config.layoutBreakpoints[0]}, ${test.config.layoutBreakpoints[1]})`)
```